packages/contracts/contracts/LenderCommitmentForwarder.sol

```diff
@@ -215,14 +229,21 @@ contract LenderCommitmentForwarder is TellerV2MarketForwarder {
      */
     function updateCommitment(
         uint256 _commitmentId,
-        Commitment calldata _commitment
+        Commitment calldata _commitment,
+        address[] calldata borrowerAddressList
     )
         public
         commitmentLender(_commitmentId)
         validateUpdatedCommitment(lenderMarketCommitments[_commitmentId])
     {
         lenderMarketCommitments[_commitmentId] = _commitment;
 
+        delete commitmentBorrowersList[_commitmentId];
```

Comment:
we should see if this actually works since the `EnumerableSet` contains an array and mapping inside which may not be cleared

Reply:
done - it does work